cve_bin_tool/OutputEngine.py

```diff
@@ -45,7 +45,7 @@ def generate_filename(self, extension=None):
 
     def output_cves(self, outfile, output_type=None):
         """ Output a list of CVEs
-        format self.modules[checker_name][version] = dict{id: severity}
+        format self.products[checker_name][version] = dict{id: severity}
```

Comment:
```suggestion
        format self.checkers[checker_name][version] = dict{id: severity}
```

We're referring to a list of checkers here, so we might as well call it that.